pkg/apis/dos/validation/dos.go

```diff
@@ -136,7 +132,10 @@ func validateAppProtectDosLogDest(dstAntn string) error {
 }
 
 func validatePort(value string) error {
-	port, _ := strconv.Atoi(value)
+	port, err := strconv.Atoi(value)
+	if err != nil {
```

Comment:
is a new unit test for this portion possible?

Reply:
It's not necessary tbh. The reason is it's a one line func from the Go std library. The `behavior` of the exported func that calls `validatePort` (in a chain of other func calls) is cover by unit tests already.

Reply:
> is a new unit test for this portion possible?

added additional tests for not exported functions